test/components/Layout_test.tsx

```diff
@@ -17,20 +17,21 @@ describe("App component", () => {
       expect(app.queryByTestId("sessionInfo-bar")).toBeNull();
     });
 
-    it("should render a session top bar when anonymous", () => {
+    it("should render a session top bar when anonymous", async () => {
       const serverInfo = {
         url: "http://test.server/v1/",
         capabilities: {},
       };
       store.dispatch(sessionActions.serverInfoSuccess(serverInfo));
       store.dispatch(sessionActions.setAuthenticated());
+      await waitFor(() => new Promise(resolve => setTimeout(resolve, 500))); // debounce wait
```

Comment:
What does this do here concretely?

Reply:
Changing authentication contexts takes a little bit of time between the ServerHistory [debounced fetch call](https://github.com/Kinto/kinto-admin/blob/master/src/components/ServerHistory.tsx#L72) which has 400ms of waiting, and redux.

It seems like with the upgrade to react 18 our test weren't waiting automatically in some cases. This should only be a problem on tests that hit `AuthForm` and `HomePage` (which includes AuthForm) without mocking session up front.